cartridges/int_cloudinary_sfra/cartridge/controllers/Product.js

```diff
@@ -106,9 +107,14 @@ server.append('Variation', function (req, res, next) {
             colorAttrValueID = params.variables.color.value;
         }
 
+        if (params && params.variables && params.variables.accessorySize && params.variables.accessorySize.value || params.variables.size && params.variables.size.value) {
```

Comment:
if params is not defined, the code will break on evaluating params.variables.size. Please use parenthesis according operators priority